sklearn/metrics/_ranking.py

```diff
@@ -112,6 +112,15 @@ def auc(x, y):
     return area
 
 
+@validate_params(
+    {
+        "y_true": ["array-like"],
```

Comment:
It is true that we are going to call `column_or_1d` that will transform any array-like into a NumPy array. Could you update the docstring of `y_true` and `y_score` to change `ndarray` by `array-like`